3.9/graphs-general-graphs-management.md

```diff
@@ -374,11 +376,13 @@ graph with different *from* and/or *to* collections an error is thrown.
 
 Modify a relation definition:
 
-`graph_module._editEdgeDefinitions(edgeDefinition)`
+`graph_module._editEdgeDefinitions(edgeDefinition, options)`
 
 - `edgeDefinition` (object):
   The edge definition to replace the existing edge definition with the same
   attribute *collection*.
+- `options` (object):
+  Additional options related to the edgeDefinition itself.
```

Comment:
Can we add a link to the Options defined later, stating what values are allowed here?
If not I think we need to copy the "satellite" sentence to here as well, otherwise it is not clear what to do from a user-perspective. Searching for "options" is kind of hopeless.

Reply:
added.